.thoth.yaml

```diff
@@ -0,0 +1,22 @@
+host: khemenu.thoth-station.ninja
+tls_verify: false
+requirements_format: pipenv
+
+runtime_environments:
+  - name: rhel:8
+    operating_system:
+      name: rhel
+      version: "8"
+    python_version: "3.6"
+    recommendation_type: latest
+
+managers:
+  - name: info
+  - name: version
+    configuration:
+      maintainers:
+        - vpavlin
```

Comment:
maintainer list allows only allowed users to make a release request to kebechet.
Please add other here who might need permission to release the project with thoth-bots.

Reply:
PLease add at least @LaVLaS we can figure out more later:)